UnityProject/Assets/Scripts/Player/PlayerSprites.cs

```diff
@@ -50,7 +50,7 @@ public class PlayerSprites : MonoBehaviour
 
 		[Tooltip("Override the race of the character sheet")]
 		[SerializeField]
-		private string raceOverride = "";
+		public string raceOverride = "";
```

Comment:
naming convention, also if we're not setting the strong from outside this you should do a `{get; private set;} = "";`

Reply:
not sure if it works with editor Inspector?

Reply:
`[field: SerilizeField]`

Reply:
I think it is finr as is

Reply:
Bod, I don't think it's negotiable what is and isn't fine.
https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/

I can maybe excuse the lack of the {get; set;}, but the naming convention is wrong, and you have a serialize field on a public variable. You're doing something wrong and there's no point in arguing about what is and isn't right when the fix is so simple and can be taken care of using your IDE.

Reply:
Is modified from outside the class playerSprites.raceOverride = requestedOccupation.CustomSpeciesOverwrite.name;

Reply:
fix the naming then and remove the attribute, please.